po/fr.po

```diff
@@ -0,0 +1,15296 @@
+msgid "Day 1: Afternoon"
+msgstr "Jour 1 : Après-midi"
+
+#: src/SUMMARY.md:44
+msgid "Variables"
+msgstr "Variables"
+
+#: src/SUMMARY.md:45
+msgid "Type Inference"
+msgstr "Inférence de type"
+
+#: src/SUMMARY.md:46
+msgid "static & const"
+msgstr ""
+
+#: src/SUMMARY.md:47
+msgid "Scopes and Shadowing"
+msgstr "Portée et ombrage"
+
+#: src/SUMMARY.md:48
+msgid "Memory Management"
+msgstr "Gestion de la mémoire"
+
+#: src/SUMMARY.md:49
+msgid "Stack vs Heap"
+msgstr "Pile et Tas"
+
+#: src/SUMMARY.md:50
+msgid "Stack Memory"
+msgstr "Mémoire de pile"
+
+#: src/SUMMARY.md:51
+msgid "Manual Memory Management"
+msgstr "Gestion manuelle de la mémoire"
+
+#: src/SUMMARY.md:52
+msgid "Scope-Based Memory Management"
+msgstr "Gestion de la mémoire basée sur la portée"
+
+#: src/SUMMARY.md:53
+msgid "Garbage Collection"
+msgstr "Collecte des ordures"
+
+#: src/SUMMARY.md:54
+msgid "Rust Memory Management"
+msgstr "Gestion de la mémoire avec Rust"
+
+#: src/SUMMARY.md:55
+msgid "Comparison"
+msgstr "Comparaison"
+
+#: src/SUMMARY.md:56
+msgid "Ownership"
+msgstr "Possession"
+
+#: src/SUMMARY.md:57
+msgid "Move Semantics"
+msgstr "Déplacer la sémantique"
+
+#: src/SUMMARY.md:58
+msgid "Moved Strings in Rust"
+msgstr "String déplacées dans Rust"
+
+#: src/SUMMARY.md:59
+msgid "Double Frees in Modern C++"
+msgstr "Erreur de la double libération de mémoire en C++ moderne"
+
+#: src/SUMMARY.md:60
+msgid "Moves in Function Calls"
+msgstr "Déplacements dans les appels de fonction"
+
+#: src/SUMMARY.md:61
+msgid "Copying and Cloning"
+msgstr "Copie et clonage"
+
+#: src/SUMMARY.md:62
+msgid "Borrowing"
+msgstr "Emprunt"
+
+#: src/SUMMARY.md:63
+msgid "Shared and Unique Borrows"
+msgstr "Emprunts partagés et uniques"
+
+#: src/SUMMARY.md:64
+msgid "Lifetimes"
+msgstr "Durées de vie"
+
+#: src/SUMMARY.md:65
+msgid "Lifetimes in Function Calls"
+msgstr "Durées de vie dans les appels de fonction"
+
+#: src/SUMMARY.md:66
+msgid "Lifetimes in Data Structures"
+msgstr "Durées de vie dans les structures de données"
+
+#: src/SUMMARY.md:68
+msgid "Designing a Library"
+msgstr "Conception d'une bibliothèque"
+
+#: src/SUMMARY.md:69
+msgid "Iterators and Ownership"
+msgstr "Itérateurs et propriété"
+
+#: src/SUMMARY.md:72
+msgid "Day 2: Morning"
+msgstr "Jour 2 : Matin"
+
+#: src/SUMMARY.md:77
+msgid "Structs"
+msgstr "Structures"
+
+#: src/SUMMARY.md:78
+msgid "Tuple Structs"
+msgstr "Structures de tuple"
+
+#: src/SUMMARY.md:79
+msgid "Field Shorthand Syntax"
+msgstr "Syntaxe abrégée des champs"
+
+#: src/SUMMARY.md:80
+msgid "Enums"
+msgstr "Énumérations"
+
+#: src/SUMMARY.md:81
+msgid "Variant Payloads"
+msgstr "Charges variantes"
+
+#: src/SUMMARY.md:82
+msgid "Enum Sizes"
+msgstr "Tailles d'énumération"
+
+#: src/SUMMARY.md:84
+msgid "Method Receiver"
+msgstr "Récepteur de méthode"
+
+#: src/SUMMARY.md:85 src/SUMMARY.md:160 src/SUMMARY.md:195
+msgid "Example"
+msgstr "Exemple"
+
+#: src/SUMMARY.md:86
+msgid "Pattern Matching"
+msgstr "Correspondance de modèle"
```

Comment:
Since model has a similar meaning as in regexp, I'd rather use "motif" than "modèle"